lib/tasks/create_bus_stop_fields.rake

```diff
@@ -0,0 +1,286 @@
+# frozen_string_literal: true
+
+namespace :bus_stop_fields do
+  task create: :environment do
+    fields = {
+      'Info' => [
+        {
+          name: 'Completed',
+          column: :completed,
```

Comment:
What is "column" in this context?

Reply:
The bus stops table column from which the value should be pulled in order to populate the bus stop field.

Reply:
Ohhh